custom-tests.yaml

```
@@ -1513,7 +1513,7 @@ api:
       <%api.RTCDataChannel:channel%>
       var instance = new RTCDataChannelEvent('', {channel: channel});
   RTCDTMFToneChangeEvent:
-    __base: "new RTCDTMFToneChangeEvent('', {tone: ''});"
+    __base: var instance = new RTCDTMFToneChangeEvent('tonechange', {tone: ''});
```

Ha, this actually does need the quotes because of the `{` and/or `:` characters in here:

```suggestion
    __base: "var instance = new RTCDTMFToneChangeEvent('tonechange', {tone: ''});"
```